src/Chr.Avro.Codegen/Codegen/CSharpCodeGenerator.cs

```diff
@@ -379,5 +388,24 @@ private static IEnumerable<NamedSchema> GetCandidateSchemas(Schema schema, ISet<
 
             return seen.OfType<NamedSchema>();
         }
+
+        private AttributeListSyntax[] GetDescriptionAttribute(string? documentation)
+        {
+            if (string.IsNullOrEmpty(documentation) || !enableDescriptionAttributeForDocumentation)
+            {
+                return Array.Empty<AttributeListSyntax>();
+            }
+
+            // Generates: [Description("documentation")]
+            // https://stackoverflow.com/questions/35927427/how-to-create-an-attributesyntax-with-a-parameter
+            var name = SyntaxFactory.ParseName("System.ComponentModel.DescriptionAttribute");
+            var arguments = SyntaxFactory.ParseAttributeArgumentList("(\"" + documentation + "\")");
```

Comment:
Good catch, you are absolutely right!

The old version actually didn't break on code generation, but the quoting was wrong:

``` csharp
/// <summary>Property "Description" with double quotes</summary>
[System.ComponentModel.DescriptionAttribute("Property "Description" with double quotes")]
public string DescriptionPropertyWithDoubleQuotes { get; set; }

// Fails on compilation
```

I wrote a test on this scenario, and the new test project caught it, since it didn't compile 👌

Your suggestion worked like a charm ✨